internal/controller/limiter/limiter.go

```diff
@@ -130,14 +131,18 @@ func (l *MaxInFlight) OnAdd(obj any, inInitialList bool) {
 		// After sync is finished, the limiter handler takes tokens directly.
 		return
 	}
+
 	// Before sync is finished: we're learning about existing jobs, so we should
 	// (try to) take tokens for unfinished jobs started by a previous controller.
 	// If it's added as already finished, no need to take a token for it.
 	// Otherwise, try to take one, but don't block (in case the stack was
 	// restarted with a different limit).
 	if !model.JobFinished(job) {
 		l.tryTakeToken("OnAdd")
-		l.logger.Debug("existing not-finished job discovered", zap.Int("tokens-available", len(l.tokenBucket)))
+		l.logger.Debug("existing not-finished job discovered",
+			zap.String("uuid", job.Labels[config.UUIDLabel]),
```

Comment:
should it be called job uuid?

Reply:
Done. For consistency I've changed it everywhere a job UUID field is logged.